### ISSUE TITLE
Add shortcut controls

### ISSUE THREAD

**Issue by philipbelesky** [mariuszhermansdorfer/SandWorm#108]
Relates to draft PR #103, i'll reproduce the intent documented there: 

>  The use case here is both ease of access and a better ability to run things 'headless' - e.g. with the controlling computer more out of the way.
> 
> In my case I have a small numpad I'd imagined for this purpose, and might consider something like a StreamDeck for exhibition-like contexts. The keys on either of those options can be remapped to whatever, so the use of F10-F20 below is relatively arbitrary and could be made to be either more intuitive or more obscure.

I was filming a few videos of the setup the other day and this would have made things a lot easier!

**Comment by mariuszhermansdorfer** [comment]
Makes a lot os sense. Ideally, user could define the key combinations. That's relatively easy to do in Rhino, but I don't have a good idea on how to approach it in GH. Any suggestions?

**Comment by philipbelesky** [comment]
The best way I can think of would be to do some sort of pop-up modal interface that allows people to enter the remappings. That might be a bit too much effort for a relatively obscure feature though.

**Comment by mariuszhermansdorfer** [comment]
Agreed. Sounds like too much effort for the value it might potentially add.

**Comment by mariuszhermansdorfer** [comment]
Done in https://github.com/mariuszhermansdorfer/SandWorm/commit/d042c002168acb854b4e8f00aa24ba3a12561d91

**Comment by philipbelesky** [comment]
Possibly of interest: a little demo of how a numberpad with dials can be used as a controller

https://user-images.githubusercontent.com/495961/174000898-4490d366-30e9-4f32-b6ee-8291a6b01f2c.mov

**Comment by mariuszhermansdorfer** [comment]
Looks great! How much do these cost?

**Comment by philipbelesky** [comment]
For ones with dials they tend to be around 50-75USD, although may require some assembly or simple soldering. They tend to go in small batches or runs though - searching for a "macro pad" should bring up options. Alternately, a standard numberpad with bluetooth can also be found pretty cheap.
